vxvas2nets/vas2nets_sms.py

```diff
@@ -179,19 +219,42 @@ def handle_outbound_message(self, message):
             returnValue((yield self.handle_outbound_fail(message, resp)))
 
     @inlineCallbacks
+    def handle_send_timeout(self, message):
+        yield self.add_status(
```

Comment:
A RabbitMQ `nack` would cause a retry, but I meant a Vumi `nack` (which would just let the application know that we don't think a message was successfully delivered to a third party).